cumulus_library_opioid/vocab/common/keywords/keywords.curated.sql

```diff
@@ -1,4 +1,4 @@
--- Its unclear where RXNCONSO_curated comes from
+-- Its unclear where all_rxcui_str.RXNCONSO_curated comes from
```

Comment:
What does this mean exactly?

Reply:
@comorbidity has some table locally in the mysql version and i'm not sure where it is generated from.